src/main/frontend/components/shortcut.cljs

```diff
@@ -58,12 +66,12 @@
          [:thead
           [:tr
            [:th.text-left [:b (t name)]]
-           [:th.text-right [:b (t :help/shortcut)]]]]
+           [:th.text-right]]]
```

Comment:
I removed the text here because it doesn't add anything and is just a little confusing imo.